### PR TITLE
[RFC] Add docs for the ThemeManager

### PR DESCRIPTION
**Pull request by timunie**
I am not sure if I wrote everything correctly. 
1. I don't understand what is going on in the detail, so I might have missed something 
2. I am not a native English speaker, so if someone is a native speaker feel free to report spelling errors.

### CODE REVIEW

**Comment by timunie** [comment]
Tasks to do: 
[ ] Add documentation to Theme Syncing

**Comment by batzen** [comment]
@timunie just ping me if you want me to have a look. Don't want to disturb you while you are still working on it.

**Comment by timunie** [comment]
Hi @batzen 

Does it look so bad? This is all I know about ThemeManager. I already got people who understood it and were able to do the Theming. 

If you can give me a feedback about what to add I would be happy to do so.

**Comment by batzen** [comment]
I didn't mean to say it's bad, sorry if it sounded like that. I just wanted to offer help but didn't want to interrupt your work with review comments. Will try to add some comments in the next few days. I wrote most of the code and even i discovered new usage scenarios while using it at work. 😂

**Comment by timunie** [comment]
I look forward to ylur comments or if you like it more direct changes.

File: Wiki/ThemeManager.md [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+using ControlzEx.Theming;
+```
+
+now you can apply the `Theme` of your choice to your `App`, to a specific `Window` or any `FrameworkElement`. Just call this line in your code: 
+```c#
+ThemeManager.Current.ChangeTheme([Affected item], "[Name of the theme]");
+```
+
+Where `[Affected item]` may be:
+- `Applictaion.Current`
```
Comment (seba30) [inline Wiki/ThemeManager.md:40]: Typo, should be **Application**

File: Wiki/ThemeManager.md [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+    ```c#
+    ThemeManager.Current.AddTheme(newTheme);
+    ```
+    
+5. Apply new `Theme`
+    ```c#
+    ThemeManager.Current.ChangeTheme([Affected item], newTheme);
+    ```
+    Where `[Affected item]` may be:
+    - `Applictaion.Current`
```
Comment (seba30) [inline Wiki/ThemeManager.md:136]: Typo, should be **Application**

File: Wiki/ThemeManager.md [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+```c#
+using ControlzEx.Theming;
+```
+
+Now you can call this statement to get the current theme:
+```c#
+Theme currentTheme = ThemeManager.Current.DetectTheme([ThemedItem]);
+```
+where `ThemedItem` may be:
+- `Applictaion.Current`
```
Comment (seba30) [inline Wiki/ThemeManager.md:279]: Typo, should be **Application**

**Review by seba30 — CHANGES_REQUESTED** [review]
Never done this before.

File: Wiki/ThemeManager.md (hunk 2 above)
Comment (timunie) [reply Wiki/ThemeManager.md]: thx :pray:

File: Wiki/ThemeManager.md (hunk 3 above)
Comment (timunie) [reply Wiki/ThemeManager.md]: thx :pray:

File: Wiki/ThemeManager.md (hunk 1 above)
Comment (timunie) [reply Wiki/ThemeManager.md]: thx :pray:

**Comment by batzen** [comment]
@timunie Thanks for contributing this!!!

**Pull request merged**